core/oputil/oputil_ssh_key.js

```diff
@@ -0,0 +1,158 @@
+const _ = require('lodash');
+
+
+exports.handleSSHKeyCommand = handleSSHKeyCommand;
+
+const ConfigIncludeKeys = [
+    'loginServers.ssh',
+    'loginServers.ssh.privateKeyPem',
+];
+
+const MINIMUM_PASSWORD_LENGTH = 8;
+const QUESTIONS = {
+    Create: [
+        {
+            name: 'createNew',
+            message: 'Generate New SSH Keys?',
+            type: 'confirm',
+            default: false,
+        },
+        {
+            name: 'password',
+            message: 'SSL Password:',
+            default: "",
+            when: answers => answers.createNew,
+        },
+    ],
+};
+
+function execute(ui, command) {
+    ui.log.write("Ping!");
+    ui.log.write(command);
+    exec(
+        command,
+        function (error, stdout, stderr) {
+            ui.log.write(error);
+
+            if (error) {
+                const reason = error ? error.message : 'OpenSSL Error';
+                logDebug(
+                    {
+                        reason: reason,
+                        cmd: util.cmd,
+                        args: args
+                    },
+                    `openssl command failed`
+                );
+            }
+            else {
+                ui.log.write("SSH Keys Generated")
+            }
+        }
+    );
+}
+
+function createNew(cb) {
+    const ui = new inq.ui.BottomBar();
+
+    let sslPassword;
+
+    async.waterfall(
+        [
+            function init(callback) {
+                return initConfigAndDatabases(callback);
+            },
+            function create(configuration, callback) {
+                getAnswers(QUESTIONS.Create, answers => {
+                    if (!answers.createNew) {
+                        return callback('exit');
+                    }
+
+                    // Get Answer Value
+                    sslPassword = answers.password;
+                    if (!sslPassword || sslPassword.replaceAll(" ", "") == "") {
+                        ui.log.write('Password must be set.');
+
+                        return callback('exit');
+                    }
+                    if (sslPassword.length < MINIMUM_PASSWORD_LENGTH) {
+                        ui.log.write(`Password must be at least ${MINIMUM_PASSWORD_LENGTH} characters.`);
+
+                        return callback('exit');
+                    }
+
+                    // Check if Keyfiles Exist
+                    const sshKeyPath = "config/security/";
+                    const sshKeyFilename = "ssh_private_key.pem";
+                    const targetKeyFile = sshKeyPath + sshKeyFilename;
+
+                    // Check if Keyfile Exists
+                    if (fs.existsSync(targetKeyFile)) {
+                        ui.log.write(`${targetKeyFile} already exists.`)
+
+                        return callback('exit');
+                    }
+
+                    ui.log.write(`Creating SSH Key: ${targetKeyFile}`);
+
+                    // Create Dir
+                    if (!fs.existsSync(sshKeyPath)) {
```

Comment:
Please use the fs-extra module, it's already a dep. Same thing here, no need to check if it exists, just try and ignore if it does exist.

https://github.com/jprichardson/node-fs-extra/blob/HEAD/docs/ensureDir-sync.md

Reply:
Done